addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4171,22 +4193,28 @@ registry.sizing = SnippetOptionWidget.extend({
      * @override
      */
     start: function () {
-        var self = this;
-        var def = this._super.apply(this, arguments);
+        const self = this;
+        const def = this._super.apply(this, arguments);
 
         this.$handles = this.$overlay.find('.o_handle');
 
-        var resizeValues = this._getSize();
+        let resizeValues = this._getSize();
         this.$handles.on('mousedown', function (ev) {
             ev.preventDefault();
 
+            // If the handle has the class 'readonly', don't allow to resize.
+            // (For the grid handles when we are in mobile view).
+            if (this.classList.contains('readonly')) {
```

Comment:
I don't like the use of `this` in jQuery handlers, source of error when we'll convert to no jquery or simply if someone convert to function to an arrow function. Use `ev.currentTarget` (actually is not `this` `ev.target`? checking target instead of currentTarget would be wrong)